services/movie_session.py

```diff
@@ -0,0 +1,48 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        cinema_hall_id: int,
+        movie_id: int
+) -> QuerySet:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id
+    )
+
+
+def get_movies_sessions(
+        session_date: str = None
+) -> QuerySet:
+    if session_date:
+        return MovieSession.objects.filter(show_time__date=session_date)
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(movie_session_id: int) -> QuerySet:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+        session_id: int,
+        show_time: datetime.date = None,
+        movie_id: int = None,
+        cinema_hall_id: int = None
+) -> None:
+    movie_session = MovieSession.objects.filter(id=session_id)
+    if show_time:
+        movie_session.update(show_time=show_time)
+    if movie_id:
+        movie_session.update(movie_id=movie_id)
+    if cinema_hall_id:
+        movie_session.update(cinema_hall_id=cinema_hall_id)
+
+
+def delete_movie_session_by_id(session_id: int) -> QuerySet:
```

Comment:
You could indicate a `QuerySet` of what instances returned in the function here and in the other functions in the return type annotation.

Reply:
you mean "couldn`t"?Because,i already did indicate return type annotation for all functions with QuerySet